Description: Since Ubuntu's gnome-screenshot doesn't automatically save to the
 Pictures directory, don't claim that it does in the Keyboard Shortcuts list
Author: William Hua <william.hua@canonical.com>
--- a/panels/keyboard/01-screenshot.xml.in
+++ b/panels/keyboard/01-screenshot.xml.in
@@ -1,17 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <KeyListEntries group="system" schema="org.gnome.settings-daemon.plugins.media-keys" _name="Screenshots">
 
-	<!-- translators: Pictures is the name of the XDG Pictures directory -->
 	<KeyListEntry name="screenshot"
-		_description="Save a screenshot to Pictures"/>
+		_description="Take a screenshot"/>
 
-	<!-- translators: Pictures is the name of the XDG Pictures directory -->
 	<KeyListEntry name="window-screenshot"
-		_description="Save a screenshot of a window to Pictures"/>
+		_description="Take a screenshot of a window"/>
 
-	<!-- translators: Pictures is the name of the XDG Pictures directory -->
 	<KeyListEntry name="area-screenshot"
-                      _description="Save a screenshot of an area to Pictures"/>
+                      _description="Take a screenshot of an area"/>
 
 	<KeyListEntry name="screenshot-clip"
                       _description="Copy a screenshot to clipboard"/>
